docs/en/sql-reference/table-functions/mysql.md

```diff
@@ -81,8 +81,8 @@ SELECT * FROM mysql('localhost:3306', 'test', 'test', 'bayonet', '123');
 Replacing and inserting:
 
 ```sql
-INSERT INTO FUNCTION mysql('localhost:3306', 'test', 'test', 'bayonet', '123', 1) ('int_id', 'float') VALUES (1, 3);
-INSERT INTO TABLE FUNCTION mysql('localhost:3306', 'test', 'test', 'bayonet', '123', 0, 'UPDATE int_id = int_id + 1') ('int_id', 'float') VALUES (1, 4);
+INSERT INTO FUNCTION mysql('localhost:3306', 'test', 'test', 'bayonet', '123', 1) (int_id, float) VALUES (1, 3);
+INSERT INTO TABLE FUNCTION mysql('localhost:3306', 'test', 'test', 'bayonet', '123', 0, 'UPDATE int_id = int_id + 1') (int_id, float) VALUES (1, 4);
 SELECT * FROM mysql('localhost:3306', 'test', 'test', 'bayonet', '123');
 ```
 
```

Comment:
@tavplubix Not good, IMHO. In the **Examples** section we give just some examples of how it works, not all the possible cases of correct syntax.

Reply:
I've just fixed incorrect syntax, column names list is a list of identifiers, not string literals (the was an error in my example in previous comment, sorry)

Reply:
> Will the result of the last SELECT query be the same after two more INSERTs?

The are only two inserts, as before.

Reply:
@tavplubix Oh sorry, I misunderstood!